charts/clickhouse-serverless/tests/e2e.sh

```
@@ -0,0 +1,211 @@
+#   CLUSTER_NAME            — Kind cluster name (default: ch-test)
+#   TIMEOUT                 — helm --wait timeout in seconds (default: 480)
+#   TEST_REPLICATED=true    — also run the 3-node replicated suite
+
+set -euo pipefail
+
+CLUSTER_NAME="${CLUSTER_NAME:-ch-test}"
+RELEASE="ch"
+NAMESPACE="ch-test"
+CHART_DIR="$(cd "$(dirname "$0")/.." && pwd)"
+TIMEOUT="${TIMEOUT:-480}"
+CH_PASSWORD="e2etest"
```

Shellcheck (0.11.0)

[warning] CH_PASSWORD appears unused. Verify use (or export if used externally).

(SC2034)